storage/src/vespa/storage/bucketdb/bucketinfo.h

```diff
@@ -21,6 +21,7 @@ template <typename NodeSeq>
 class BucketInfoBase
 {
 protected:
+    //TODO: Should we use a chrono timepoint to ensure we are using same clock everywhere ?
```

Comment:
It would be good to unify this, but I think we'd have to be careful to not expose an API that makes it seem we can store and later retrieve a higher resolution time point than seconds. We're limited to u32 seconds after epoch, since the GC timestamp is stored as the 32 MSBs of the 64-bit bucket DB B-tree KV value (with a DataStore entry ref taking up the 32 LSBs).

Reply:
Agree